thimport.cxx

```diff
@@ -41,10 +41,30 @@
 
 #include <fmt/printf.h>
 
+constexpr auto ANON_STATION_NAME = "-";
+
+/**
+ * Helper function for converting a string vector to a vector of char pointers.
+ */
+static std::vector<char *>
+_string_vec_to_charp_vec(std::vector<std::string> & svec) {
```

Comment:
Please remove prefix `_`, because global identifiers with such prefix are reserved by the C++ standard, CI fails on this.